src/Admin/SupportScreen.php

```diff
@@ -163,13 +165,11 @@ public function enqueue_assets( $hook_suffix ) {
 		);
 
 		$args    = [];
-		$post_id = isset( $_GET['post_id'] ) ? (int) $_GET['post_id'] : 0; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$amp_url = isset( $_GET['url'] ) ? esc_url_raw( $_GET['url'] ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 
-		if ( ! empty( $post_id ) && 0 < intval( $post_id ) ) {
+		if ( ! empty( $amp_url ) ) {
```

Comment:
@dhaval-parekh Maybe we could validate here if a given string is actually an url? Something like:

```suggestion
		if ( ! empty( $amp_url ) && filter_var( $amp_url, \FILTER_VALIDATE_URL ) ) {
```